app/assets/javascripts/sephora_style_guide/video.js

```diff
@@ -60,4 +49,19 @@ function onPlayerReady() {
       }
     }
   });
+
+  $(document).on('keyup', '.modal[role="dialog"]', function(e) {
```

Comment:
Why not use events that are already provided by bootstrap? https://getbootstrap.com/docs/4.1/components/modal/#events

`hide.bs.modal` looks like the appropriate one if you ask me.